teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher
```

Comment:
- [ ] As the previous reviewer mentioned, kindly make the Teacher class inherit from the person class. 
```
class Teacher < Person
```